src/components/gameAnalysis/styles/teamStats.s.tsx

```diff
@@ -27,8 +32,25 @@ export const TeamStat = styled.div`
     border-right: 2px solid ${Palette.GG_STATS_BORDER};
   }
 `;
+const gg_svg: {
```

Comment:
https://stackoverflow.com/questions/54438012/an-index-signature-parameter-type-cannot-be-a-union-type-consider-using-a-mappe